hack/kubearchive-delete.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o xtrace
+
+YAML=$(mktemp --suffix=.yaml -t kubearchive-XXX)
+
+helm template kubearchive charts/kubearchive -n kubearchive \
+    --include-crds \
+    --set "global.production=true" > ${YAML}
+kubectl delete -n kubearchive -f ${YAML}
+
+rm -f ${YAML}
```

Comment:
What about removing the whole namespace for deleting kubearchive?

```suggestion
kubectl delete ns kubearchive
```

Reply:
Deleting the namespace won't delete any cluster-level resources that got installed.

And the namespace is getting deleted anyway as `global.production` is set to `true`.

Reply:
Sorry for reopening this thread but I have another question related to this script:

We are removing the resources that are created from the helm chart, but in this `DEVELOPMENT` instructions we apply other test resources like the `log-generators` or the `test` users I'm introducing in https://github.com/kubearchive/kubearchive/pull/456. When "deleting" kubeArchive, should we also consider removing these stuff? Or maybe not because they aren't kubeArchive components?

It's probably ok this way, but I just wanted to raise my concern so you could also think about it.

Reply:
IMO those things are completely separate from KubeArchive.